src/otx/v2/adapters/torch/mmengine/mmdeploy/exporter.py

```diff
@@ -85,7 +85,9 @@ def _get_inputs(self) -> tuple[torch.Tensor, dict | None]:
         """Prepare torch model's input and input_metas."""
         input_shape = self.deploy_cfg.backend_config.model_inputs[0]["opt_shapes"]["input"]
         input_tensor = torch.randn(input_shape)
-        input_metas = None
+        input_metas = self.deploy_cfg.backend_config.get("input_metas", {})
+        input_metas["mode"] = "predict"
+        input_metas["data_samples"] = input_tensor
```

Comment:
```suggestion
        input_metas = self.deploy_cfg.backend_config.get("input_metas", None)
        if input_metas is None:
            input_metas = {"data_samples": [BaseDataElement()] * input_shape[0], "mode": "predict"}
```
You may import BaseDataElement from mmengine.structures